yarn-project/pxe/src/pxe_service/pxe_service.ts

```diff
@@ -875,72 +880,37 @@ export class PXEService implements PXE {
   }
 
   public async getPrivateEvents<T>(
+    contractAddress: AztecAddress,
     eventMetadataDef: EventMetadataDefinition,
     from: number,
-    limit: number,
-    // TODO (#9272): Make this better, we should be able to only pass an address now
-    vpks: Point[],
+    numBlocks: number,
+    recipients: AztecAddress[],
```

Comment:
Now instead of decrypting the events here we just query private event data provider.